src/app/core/export/export-service/export.service.ts

```diff
@@ -91,14 +95,32 @@ export class ExportService {
 
     const flattenedExportRows: ExportRow[] = [];
     if (config) {
-      for (const dataRow of data) {
-        const extendedExportableRows = await this.generateExportRows(
-          dataRow,
-          config,
-          from,
-          to
-        );
-        flattenedExportRows.push(...extendedExportableRows);
+      if (groupByProperty) {
```

Comment:
So it is technically possible to extend the service with this new feature, however it just becomes more and more complex.